development/compiling/compiling_for_osx.rst

```diff
@@ -53,9 +63,14 @@ editor binary built with ``scons p=osx target=release_debug``::
 
     cp -r misc/dist/osx_tools.app ./Godot.app
     mkdir -p Godot.app/Contents/MacOS
-    cp bin/godot.osx.tools.64 Godot.app/Contents/MacOS/Godot
+    cp bin/godot.osx.tools.universal Godot.app/Contents/MacOS/Godot
     chmod +x Godot.app/Contents/MacOS/Godot
 
+If you are building the ``master`` branch, additionally copy Vulkan library:
+
+    mkdir -p Godot.app/Contents/Frameworks
+    cp {VULKAN_SDK_PATH}/macOS/libs/libMoltenVK.dylib Godot.app/Contents/Frameworks/libMoltenVK.dylib
```

Comment:
Is `{VULKAN_SDK_PATH}` an env variable here (and might thus lack `$`), or a placeholder for the user to fill?
If it's a placeholder, I'd suggest using a format that is not so reminiscent of an env varibale, e.g. `<Vulkan SDK path>/macOS/...`.

Reply:
placeholder,